test/integration/dfuzzer/main.fmf

```diff
@@ -0,0 +1,31 @@
+summary: Run dbus-broker under sanitizers
+test: ./test.sh
+require:
+    - clang
+    - compiler-rt
+    - coreutils
+    - dbus
+    - dbus-devel
+    - dfuzzer
+    - expat-devel
+    - gcc-c++
+    - gdb
+    - gettext-devel
+    - git
+    - glib2-devel
+    - glibc-devel
+    - gobject-introspection-devel
+    - gtk-doc
+    - libasan
+    - libubsan
+    - llvm
+    - meson
+    - meson
```

Comment:
```suggestion
```

Reply:
That's a backup meson, just in case the first one doesn't work :)